src/qz/common/Constants.java

```diff
@@ -34,6 +34,7 @@
     public static final String ABOUT_TITLE = "QZ Tray";
     public static final String ABOUT_URL = "https://qz.io";
     public static final String ABOUT_COMPANY = "QZ Industries, LLC";
+    public static final String ABOUT_VERSION_URL = "https://github.com/qzind/tray/releases";
 
```

Comment:
@lukas-w I think we can internally control the update button on whether or not this URL is empty.  Can you handle it white-label side?

Reply:
Disregard, I've done it.